tests/tests.uischema

```diff
@@ -709,6 +709,15 @@
         "*"
       ],
       "subtitle": "Activity received"
+    },
+    "trigger": {
+      "label": "Activities (Activity received)",
+      "order": 5.1,
+      "submenu": {
+        "label": "Activities",
+        "placeholder": "Select an activity type",
+        "prompt": "Which activity type?"
+      }
```

Comment:
Pretty sure all of these were not caused by your changes.  Where did this come from?